polaris-react/src/components/TextField/TextField.scss

```diff
@@ -120,9 +144,20 @@ $spinner-icon-size: 12px;
   caret-color: var(--p-color-text);
   color: var(--p-color-text);
 
+  #{$se23} & {
+    min-height: 32px;
+    // Safari: date field alignment
+    display: flex;
+    align-items: center;
+  }
+
   @media #{$p-breakpoints-md-up} {
     font-size: var(--p-font-size-100);
     line-height: var(--p-font-line-height-2);
+
+    #{$se23} & {
+      font-size: var(--p-font-size-80-experimental);
+    }
```

Comment:
`md-down` is 16px font size, `md-up` is 13px.

### ⚠️ Design Questions: 
1. Is the input text the only font size that changes or should the labels change as well?
2. Plain buttons are `12px` (see screenshot `Action` button) which is 1px smaller than the 13px label. is this expected?
3. Should I increase the field height on mobile? Right now it is still `32px`

<img width="200" alt="Screenshot 2023-06-02 at 4 10 17 PM" src="https://github.com/Shopify/polaris/assets/20652326/bc94088a-33a4-4684-8bff-faec2a1f1f8b">

cc: @sarahill

Reply:
> 1. Is the input text the only font size that changes or should the labels change as well?
Just the input text for now. We're looking holistically at mobile web so I think the label size will change but we can do a follow up issue if needed.

> 2. Plain buttons are 12px (see screenshot Action button) which is 1px smaller than the 13px label. is this expected?
Yea for now. Same as above, still exploring mobile.

> 3. Should I increase the field height on mobile? Right now it is still 32px
Should be `36px`. I forgot to note for the input text on mobile I'm using `16px` size with `24px` line-height. If you maintain the spacing and increase the line-height that'll give use the `36px`. If we don't want to increase the line-height right now let me know and we can adjust padding accordingly.

> 4. Should the disable label be disabled when the field is disabled?
Yes